### PR TITLE
Update kestrel.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Address preliminary doc portion of https://github.com/dotnet/aspnetcore/issues/14134

@Daniel15 please review.

### CODE REVIEW

File: aspnetcore/fundamentals/servers/kestrel.md [hunk 1]
```diff
@@ -2709,6 +2709,7 @@ Host Filtering Middleware is disabled by default. To enable the middleware, defi
 
 ## Additional resources
 
+* When targeting linux, app shut down and restart may fail to restart UNIX socket. For more information, see [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/14134).
```
Comment (Daniel15) [inline aspnetcore/fundamentals/servers/kestrel.md]: I'd suggest something like this, and perhaps also include instructions on how to work around the issue:

```suggestion
* When using UNIX sockets on Linux, the socket is not automatically deleted on app shut down. For more information, see [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/14134).
```

**Comment by Daniel15** [comment]
Is this bug going to be fixed in a future ASP.NET release? The docs should probably mention that if so.

**Comment by Rick-Anderson** [comment]
@Daniel15  Thanks for the suggestion.

> Is this bug going to be fixed in a future ASP.NET release?

@Tratcher  will need to answer that

**Comment by Tratcher** [comment]
We're looking at it for 5.0.

**Comment by Rick-Anderson** [comment]
@Tratcher  Are you OK with this PR?

**Comment by Tratcher** [comment]
@anurse, I'm going dark.

**Comment by anurse** [comment]
I'll take a look.

**Pull request merged**